Do-Good-Get-Good/do-good-get-good
# Clean up jest mocks

In this PR I've cleaned up a few mocks and made them global.
Also made a few minor adjustments to make the code streamlined.

I also updated this component and its corresponding test due to there being a bug with the date handling.
`src/components/ChartCard/ChatCardDate.tsx`

### How to test
Make sure the correct date is shown in the chat card page.
Also make sure the tests don't fail.
